app.js

```diff
@@ -44,25 +47,74 @@ app.Views.TodoAppView = Backbone.View.extend({
         this.todoContent.append(this.subviews.todoListView.render());
 
         return this.$el;
+    },
+    createTodoOnEnter: function(e) {
+        const title = this.newTodoField.val().trim();
+        if (e.which == 13 && title != "") {
```

Comment:
There is almost never a case in Javascript to use double equals. Always favour triple comparisons.